modules/database/src/std/rec/waveformRecord.c

```diff
@@ -124,6 +124,7 @@ static long process(struct dbCommon *pcommon)
     struct waveformRecord *prec = (struct waveformRecord *)pcommon;
     wfdset *pdset = (wfdset *)(prec->dset);
     unsigned char pact=prec->pact;
+	epicsUInt32 nord = prec->nord;
```

Comment:
Please use 4 spaces for indentation instead of tab char's. Applies to the other changes below as well.